Tasks/Maven/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -42,13 +42,19 @@
   "loc.input.label.sqAnalysisEnabled": "Run SonarQube Analysis",
   "loc.input.help.sqAnalysisEnabled": "Run a [SonarQube analysis](http://go.microsoft.com/fwlink/?LinkID=708598) after executing the current goals. 'install' or 'package' goals should be executed first.",
   "loc.input.label.sqConnectedServiceName": "SonarQube Endpoint",
-  "loc.input.help.sqConnectedServiceName": "The endpoint that specifies the SonarQube server to use",
+  "loc.input.help.sqConnectedServiceName": "The SonarQube server generic endpoint",
   "loc.input.label.sqDbDetailsRequired": "The SonarQube server version is lower than 5.2",
   "loc.input.help.sqDbDetailsRequired": "If using a SonarQube server 5.1 or lower, you must specify the database connection details.",
   "loc.input.label.sqDbUrl": "Db Connection String",
   "loc.input.help.sqDbUrl": "SonarQube server 5.1 and lower only. Enter the database connection setting (i.e. sonar.jdbc.url). For example: jdbc:jtds:sqlserver://localhost/sonar;SelectMethod=Cursor",
   "loc.input.label.sqDbUsername": "Db Username",
   "loc.input.help.sqDbUsername": "SonarQube server 5.1 and lower only. Enter the username for the database user (i.e. sonar.jdbc.username).",
   "loc.input.label.sqDbPassword": "Db User Password",
-  "loc.input.help.sqDbPassword": "SonarQube server 5.1 and lower only. Enter the password for the database user i.e. sonar.jdbc.password"
+  "loc.input.help.sqDbPassword": "SonarQube server 5.1 and lower only. Enter the password for the database user i.e. sonar.jdbc.password",
+  "loc.input.label.pmdAnalysisEnabled": "Run PMD Analysis",
+  "loc.input.help.pmdAnalysisEnabled": "Use the PMD Java static analysis tool to look for bugs in the code.",
```

Comment:
Ah - thanks very much. This must have been lost in the middle of all the jumping between branches last week.